service/rtc/session.go

```diff
@@ -99,14 +112,47 @@ func (s *session) getScreenStreamID() string {
 	return s.screenStreamID
 }
 
-func (s *session) getRemoteScreenTrack() *webrtc.TrackRemote {
+func (s *session) getRemoteScreenTrack(rid string) *webrtc.TrackRemote {
 	s.mut.RLock()
 	defer s.mut.RUnlock()
-	return s.remoteScreenTrack
+
+	if rid == "" {
+		rid = SimulcastLevelDefault
+	}
+
+	return s.remoteScreenTracks[rid]
+}
+
+func (s *session) getRateMonitor(rid string) *RateMonitor {
+	s.mut.RLock()
+	defer s.mut.RUnlock()
+
+	if rid == "" {
+		rid = SimulcastLevelDefault
+	}
+
+	return s.screenRateMonitors[rid]
+}
+
+func (s *session) getOutScreenTrack(rid string) *webrtc.TrackLocalStaticRTP {
+	s.mut.RLock()
+	defer s.mut.RUnlock()
+	return s.outScreenTracks[rid]
```

Comment:
The way it's currently used we don't because it's called in here:

https://github.com/mattermost/rtcd/blob/823fbd14146fce1a5ac7ec7980ac972064936209/service/rtc/simulcast.go#L166-L177

And this only applies in case of simulcast so the RID cannot be empty as `getSimulcastLevel()` will always return something.